### DISCUSSION TITLE
Ctrl+E export not assigned on darkroom

### DISCUSSION THREAD

**Discussion by bigbasou** [aurelienpierreeng/ansel#125] [Q&A]
When I am on Darkroom and i want o export only one pohto I have  the messahe CTRL+e not assigned
I am not able to add this shortcut  someboby can help me please
Basilio

**Comment by dnlyko** [comment]
I believe you need to be in LightTable, not DarkRoom for CTRL+e to function

**Reply by bigbasou** [reply]
Thank you for your answer but if I remember correctly when I was using Darktable it was possible to do it and for me it was an ease

**Reply by dok-prog** [reply]
Yes, that is what I used to do, ctrl+e in darkroom. I had to get used to go back to the lighttable before launching the export. I did not find any other solution.

**Comment by dnlyko** [comment]
Correct, but not in Ansel.  Aurélien may have felt the extra code was not worth keeping.  I just hit 'L' and then CTRL+E

**Reply by bigbasou** [reply]
I used to use this function when I reworked a photograph. I will work differently

**Comment by lu-k** [comment]
The Export module in the left panel has been removed from Darkroom in Ansel. Hence the shortcut has gone too.

@dnlyko 
The default shortcuts to go from Lighttable and Darkroom and back have changed to ESC and Enter key if I'm not mistaken. You might still have old shortcuts in your setup

@bigbasou check Help > Keyboard shortcuts
